pkg/action/upgrade_qcs6490.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright © 2022 SUSE LLC
```

Comment:
Is this a suse or a kairos maintainers one?

Reply:
This is a copypaste one xDDDD